### DISCUSSION TITLE
Use Innersum method to average values

### DISCUSSION THREAD

**Discussion by JohnJimAir** [tuneinsight/lattigo#418] [Q&A]
If I want to use Innersum method to average values, what I all need to do is to write "batch := 1, n := slot"?  Am I right?

**Comment by Pro7ech** [comment]
Dear @yiquedexianshi ,

Thanks for your question. Please, give us more context information about the use case in your setting, and whether it is intended for academic or industrial use. This information can help us in giving you a more informed response about your request, and also to shape the future development of Lattigo accounting to the requirements of the community. If you prefer, you can also send us an email to [lattigo@tuneinsight.com](mailto:lattigo@tuneinsight.com).

Best,
Jean-Philippe
